packages/schematics/angular/application/files/module-files/src/main.ts.template

```diff
@@ -1,9 +1,10 @@
 <% if(!!viewEncapsulation) { %>import { ViewEncapsulation } from '@angular/core';
+<% }%><% if(!zoneless) { %>import { provideZoneChangeDetection } from '@angular/core';
 <% }%>import { platformBrowser } from '@angular/platform-browser';
 import { AppModule } from './app/app<%= typeSeparator %>module';
 
 platformBrowser().bootstrapModule(AppModule, {
-  <% if(!zoneless) { %>ngZoneEventCoalescing: true,<% } %><% if(!!viewEncapsulation) { %>
-  defaultEncapsulation: ViewEncapsulation.<%= viewEncapsulation %><% } %>
+  <% if(!zoneless) { %> applicationProviders: [provideZoneChangeDetection({ eventCoalescing: true })], <% } %>
```

Comment:
This provider should instead go in the module alongside `provideBrowserGlobalErrorListeners`: packages/schematics/angular/application/files/module-files/src/app/app__typeSeparator__module.ts.template